src/piton/extractors/omop.py

```diff
@@ -0,0 +1,205 @@
+"""A class and program for converting OMOP v5 sources to piton."""
+
+from __future__ import annotations
+
+import dataclasses
+import datetime
+from typing import Mapping, Optional, Sequence
+
+from piton import Event
+from piton.extractors.csv import CSVExtractor
+
+OMOP_BIRTH = 4216316
```

Comment:
I see that it is being used in __init__.py file inside transforms, so why is it being defined in this file?

Reply:
Fixed by moving the code to transforms/stanford.py